README.md

```diff
@@ -64,6 +64,13 @@ const { config } = await loadConfig({});
 const { config, configFile, layers } = await loadConfig({});
 ```
 
+Resolve configuration file path:
+
+```js
+// Resolve the path to the configuration file without loading it
```

Comment:
If layers are used `resolveConfigPath > resolveSources` might also download layer which is not mentioned (also loading dotenv which is a side-effect)